### PR TITLE
feat: Updated sources; Added 13.x version

### PR DESCRIPTION
**Pull request by alexandrtovmach**

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
@nodejs/i18n

**Comment by alexandrtovmach** [comment]
@JohnTitor Of course it's generated by scripts in the repo =)

File: test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const contentDir = path.join(__dirname, 'content')
 
 test('defines nodeVersions', () => {
   const majors = Object.keys(nodeVersions)
-  const expectedMajors = ['v12.x', 'v10.x', 'v8.x']
+  const expectedMajors = ['v13.x', 'v12.x', 'v10.x', 'v8.x']
```
Comment (zeke) [inline test.js]: Drive-by comment 👋 

I think we originally discussed only doing translations for even-numbered versions of Node.js that will have LTS releases. Was that decision changed?

Comment (alexandrtovmach) [reply test.js]: Sorry, I've never seen/heard about that, so just added it. I think we should discuss this on weekly meeting, but now I'm thinking that's okay to translate each version.
I was made research about crowdin functionality, and now I'm pretty sure that translation process will be much easier than before. We'd just translate the difference between the versions (10-30 files usually), all other stuff will do automatically. So probably we'd like to change policy about even-numbered versions

Comment (zeke) [reply test.js]: Sounds good to me.

> We'd just translate the difference between the versions (10-30 files usually), all other stuff will do automatically.

Nice. Are you thinking of using Crowdin's Translation Memory feature to fill in the gaps between versions?

Comment (alexandrtovmach) [reply test.js]: Yep, TM is a best way for that

Comment (nschonni) [reply test.js]: Should 8 be removed from here now that it is EOL?

Comment (amorist) [reply test.js]: FYI  https://github.com/nodejs/i18n/pull/83  https://github.com/nodejs/i18n/pull/84

**Comment by alexandrtovmach** [comment]
@nodejs/i18n Let's merge this
